### ISSUE TITLE
DocumentationTextMustEndWithAPeriod should still fire if the text ends with ">"

### ISSUE THREAD

**Issue by RobSiklos** [StyleCop/StyleCop#98]
For the SA1629 (`DocumentationTextMustEndWithAPeriod`) rule, I notice from the StyleCop source code (https://github.com/StyleCop/StyleCop/blob/master/Project/Src/StyleCop.CSharp.Rules/DocumentationRules.cs#L1426) that there is explicit code which will prevent the rule from firing if the text ends with an XML closing tag (i.e. last character is '>').

What is the rationale for this exclusion? I can't think of any cases where it would be appropriate.  Can we remove it?

In most cases I've seen, even if the last piece of content in the documentation text is an XML element, the comment should still end with a period. Examples of where a period should be present at the end, and should be flagged if the period is missing:

```
<exception cref="ArgumentNullException">
  <paramref name="userId"/> is <see cref="Guid.Empty"/>.
</exception>

<returns>
  <see langword="true"/> if an entry with the specified name exists in the
  specified folder; otherwise <see langword="false"/>.
</returns>
```

I don't have any good examples where the lack of a period is expected. We've been operating without this logic for a few years already.

**Comment by JojoShow** [comment]
The reason for code that you mentionned and that prevent firing might when there's several lines formatted using the 'para' tag.

`/// <exception cref="ArgumentOutOfRangeException">
///     <para>First condition here.</para>
///     <para>-or-</para>
///     <para>Second codition here.</para>
/// </exception>
`

**Comment by RobSiklos** [comment]
@JojoShow Yeah, that's a good case. So then maybe the code should be updates so that the exclusion only happens if the code ends with `<para>`, rather than simply `>`
